### PR TITLE
[BEAM-2098] Walkthrough URL in example code Javadoc is 404 not found

### PR DESCRIPTION
**Pull request by bpgergo**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: examples/java/src/main/java/org/apache/beam/examples/cookbook/TriggerExample.java [hunk 1]
```diff
@@ -75,8 +75,8 @@
  *
  * <p>Before running this example, it will be useful to familiarize yourself with Beam triggers
  * and understand the concept of 'late data',
- * See: <a href="http://beam.apache.org/use/walkthroughs/">
- * http://beam.apache.org/use/walkthroughs/</a>
+ * See: <a href="https://beam.apache.org/get-started/wordcount-example/">
```
Comment (lukecwik) [inline examples/java/src/main/java/org/apache/beam/examples/cookbook/TriggerExample.java]: I think this should point to:
https://beam.apache.org/documentation/programming-guide/#triggers

**Comment by lukecwik** [comment]
R: @lukecwik

File: examples/java/src/main/java/org/apache/beam/examples/cookbook/TriggerExample.java (hunk 1 above)
Comment (bpgergo) [inline examples/java/src/main/java/org/apache/beam/examples/cookbook/TriggerExample.java]: You're right, fixed that

**Comment by lukecwik** [comment]
LGTM

**Pull request closed without merging**